### PR TITLE
Use exposure time for graphs

### PR DESCRIPTION
**Pull request by cquiroz**
This uses the actual exposure time and count for the selected mode

image

### CODE REVIEW

File: common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala [hunk 1]
```diff
@@ -126,45 +105,66 @@ object ObsQueries {
...
-            mtch.id,
-            mtch.title,
-            mtch.subtitle,
-            mtch.constraintSet,
-            mtch.status,
-            mtch.activeStatus,
-            mtch.plannedTime.execution,
-            mtch.scienceMode,
-            mtch.visualizationTime
+  extension (self: ObsEditQuery.Data.type)
```
Comment (rpiaggio) [inline common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala:108]: Maybe we can define the extension over `ObsEditQuery.Data` and make `asObsEditData` return an `ObsEditData` directly. Or do we need the `Getter`?

File: common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala [hunk 2]
```diff
@@ -126,45 +105,66 @@ object ObsQueries {
...
+        }
 
-  implicit class ProgramObservationsQueryDataOps(val self: ProgramObservationsQuery.Data.type)
-      extends AnyVal {
-    def asObsSummariesWithConstraints = queryToObsSummariesWithConstraintsGetter
-  }
+  extension (self: ITCSuccess)
+    def asFixedExposureTime = FixedExposure(self.exposures, self.exposureTime)
+
+  extension (self: ProgramObservationsQuery.Data.type)
```
Comment (rpiaggio) [inline common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala:132]: Same here

**Review by rpiaggio — APPROVED** [review]
LGTM

File: common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala (hunk 1 above)
Comment (cquiroz) [reply common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala:108]: Good idea

**Pull request merged**